pkgs/by-name/do/dove/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchzip,
+  stdenvNoCC,
+  nix-update-script,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "dove";
+  version = "2025.02.01.1";
+
+  src = fetchzip {
+    url = "https://codeberg.org/celenity/Dove/raw/tag/${finalAttrs.version}/archives/dove.zip";
+    hash = "sha256-tAIWO/I9k/TE4jyGS3vHsehZFszJp03ay/tcK+PVnLI=";
+    stripRoot = false;
+    # general.config.filename is hard-coded to mozilla.cfg in wrapFirefox, so removing this setting:
+    postFetch = ''
+      sed -i '/general.config.filename/d' $out/dove.cfg $out/prefs/dove.js
```

Comment:
I can't find this.

Reply:
it's here: https://codeberg.org/celenity/Dove/src/commit/fe9dd115f7811aadc6a8135621c621a3c4e10cdc/prefs/dove.js#L2928